scripts/views/mapboxView.js

```diff
@@ -144,6 +144,27 @@
     }
   };
 
+  // add state legislature district select
+  mapboxView.stateDistrictSelect = function(feature) {
+    if (feature) {
+      // clear district highlight
+      map.setLayoutProperty('states-house-districts-selected', 'visibility', 'none');
+      map.setLayoutProperty('states-senate-districts-selected', 'visibility', 'none');
+
```

Comment:
once you take the district outlines out of `removeSelections` you can call it here.